llvm/test/tools/llvm-mca/X86/BtVer2/unsupported-instruction.s

```diff
@@ -1,6 +1,22 @@
-# RUN: not llvm-mca -mtriple=x86_64-unknown-unknown -mcpu=btver2 %s 2>&1 | FileCheck %s
+# RUN: llvm-mca -mtriple=x86_64-unknown-unknown -mcpu=btver2 -skip-unsupported-instructions %s |& FileCheck --check-prefix=CHECK-SKIP %s
+# RUN: not llvm-mca -mtriple=x86_64-unknown-unknown -mcpu=btver2 %s |& FileCheck --check-prefix=CHECK-ERROR %s
```

Comment:
Can you please include `-timeline` option so we have coverage on how a skipped instruction is handled through that code path?

Reply:
I figured this made sense for the other views showing instructions so I included those as well as your request.